docs/synthetics/ci-cd/ci-cd-monitors.md

```diff
@@ -0,0 +1,84 @@
+title: CI/CD Monitors and Monitor Groups
+description: Information regarding the unique features CI/CD Monitors and Monitor Groups have and instructions on how to set them up
+
+While Synthetics monitors are predominantly used to monitor a website at regular intervals, it's also possible to invoke them as part of your CI/CD pipelines using the **CI/CD Monitors** and **Monitor Groups** features, so that you can run them on-demand to test the availability of your services or implement regression tests that run as soon as the new version of your website becomes available.
+
+
+
+## CI/CD Monitors
+
+When creating a Synthetics monitor, you'll see that there's an option to designate that monitor as a **CI/CD Monitor**.
+
+![Creating CI/CD Monitors](/docs/images/synthetics/cicd-monitor-creation.png)
+
+This option will also be available when editing monitors.
+
+![CI/CD Option When Editing HTTP Monitors](/docs/images/synthetics/cicd-monitor-editing-http.png)
+
+![CI/CD Option When Editing Browser Monitors](/docs/images/synthetics/cicd-monitor-editing-browser.png)
+
+You'll notice that you'll no longer be able to select the interval between automatically scheduled runs for a **CI/CD Monitor**. This is because these monitors run only when you explicitly send a run request, meaning you'll execute them on-demand and avoid cluttering their run results with potentially irrelevant automatically scheduled runs. The idea here is to use Synthetics monitors as regression tests for your projects by integrating them into your CI/CD pipeline, thus catching issues before they make it to production. To reflect this lack of a regularly scheduled interval, these monitors are marked with a special badge in the **Interval** column of the Monitors Overview table.
+
+![CI/CD Monitors in the Monitors Overview Table](/docs/images/synthetics/cicd-monitors-overview.png)
+
+
+
+## CI/CD Groups
+
+When running tests for your projects, you'll most likely want to test a number of endpoints and User Journeys. To facilitate running all the **CI/CD Monitors** you need at once, you can group them together in **CI/CD Groups**. These can be created and viewed in the Synthetics sidebar.
```

Comment:
@HSSalman "CI/CD Groups" or "CI/CD Monitor Groups" or just "Monitor Groups"? Aha, looks like we use "CI/CD Groups" a lot elsewhere. OK.